test/unit/math/prim/mat/fun/matrix_exp_multiply_test.cpp

```diff
@@ -0,0 +1,44 @@
+#include <gtest/gtest.h>
+#include <test/unit/math/prim/mat/util.hpp>
+#include <stan/math/prim/mat/fun/matrix_exp.hpp>
+#include <stan/math/prim/mat/fun/matrix_exp_multiply.hpp>
+#include <vector>
+
+template <int N, int M>
+inline void test_matrix_exp_multiply() {
+  using Eigen::Dynamic;
+  using Eigen::Matrix;
+  using stan::math::matrix_exp_multiply;
+
+  std::srand(1999);
+
+  Eigen::Matrix<double, N, N> A = Eigen::Matrix<double, N, N>::Random();
+  Eigen::Matrix<double, N, M> B = Eigen::Matrix<double, N, M>::Random();
+  Eigen::Matrix<double, Dynamic, Dynamic> A0 = A;
+
+  // brute force
+  Eigen::Matrix<double, N, N> expA = stan::math::matrix_exp(A0);
+  Eigen::Matrix<double, N, M> expAB;
```

Comment:
This can just be `expAB = expA * B` because everything's `double`.   Tests can rely on the proper behavior of other functions.

Reply:
fixed.
```c++
  // brute force
  Eigen::Matrix<double, N, M> expAB
      = stan::math::multiply(stan::math::matrix_exp(A0), B);
```